include/OpenColorIO/OpenColorTypes.h

```diff
@@ -449,11 +449,14 @@ enum GpuLanguage
     GPU_LANGUAGE_GLSL_1_2,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_1_3,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_4_0,          ///< OpenGL Shading Language
-    GPU_LANGUAGE_HLSL_DX11,         ///< DirectX Shading Language
+    GPU_LANGUAGE_HLSL_SM_5_0,       ///< HLSL Shading Language
     LANGUAGE_OSL_1,                 ///< Open Shading Language
     GPU_LANGUAGE_GLSL_ES_1_0,       ///< OpenGL ES Shading Language
     GPU_LANGUAGE_GLSL_ES_3_0,       ///< OpenGL ES Shading Language
-    GPU_LANGUAGE_MSL_2_0            ///< Metal Shading Language
+    GPU_LANGUAGE_MSL_2_0,           ///< Metal Shading Language
+
+    // Deprecated enum(s)
+    GPU_LANGUAGE_HLSL_DX11 = GPU_LANGUAGE_HLSL_SM_5_0
```

Comment:
This could actually be marked as `OCIO_DEPRECATED("This was marked as deprecated starting in v2.5, please use GPU_LANGUAGE_HLSL_SM_5_0.")`, but the macro would have to be changed as follows:
```
#if __cplusplus >= 201402L || _MSVC_LANG >= 201402L
    #define OCIO_DEPRECATED(msg) [[deprecated(msg)]]
#elif defined(__GNUC__) || defined(__clang__)
    #define OCIO_DEPRECATED(msg) __attribute__((deprecated(msg)))
#else
    #define OCIO_DEPRECATED(msg) /* unsupported on this platform */
#endif
```